src/gateways/GovNotify/index.test.js

```diff
@@ -10,4 +15,28 @@ describe("GovNotify", () => {
 
     expect(instanceOne).toEqual(instanceTwo);
   });
+
+  describe("when using production environment", () => {
```

Comment:
```suggestion
  describe("when we are not running a test server", () => {
```
What do we think about changing the description to more closely match the behaviour ?